web/core/components/workspace/settings/workspace-details.tsx

```diff
@@ -308,44 +301,7 @@ export const WorkspaceDetails: FC = observer(() => {
             </div>
           )}
         </div>
-        {isAdmin && (
-          <Disclosure as="div" className="border-t border-custom-border-100">
-            {({ open }) => (
-              <div className="w-full">
-                <Disclosure.Button as="button" type="button" className="flex w-full items-center justify-between py-4">
-                  <span className="text-lg tracking-tight">Delete Workspace</span>
-                  {/* <Icon iconName={open ? "expand_less" : "expand_more"} className="!text-2xl" /> */}
-                  {open ? <ChevronUp className="h-5 w-5" /> : <ChevronDown className="h-5 w-5" />}
-                </Disclosure.Button>
-
-                <Transition
-                  show={open}
-                  enter="transition duration-100 ease-out"
-                  enterFrom="transform opacity-0"
-                  enterTo="transform opacity-100"
-                  leave="transition duration-75 ease-out"
-                  leaveFrom="transform opacity-100"
-                  leaveTo="transform opacity-0"
-                >
-                  <Disclosure.Panel>
-                    <div className="flex flex-col gap-8">
-                      <span className="text-sm tracking-tight">
-                        The danger zone of the workspace delete page is a critical area that requires careful
-                        consideration and attention. When deleting a workspace, all of the data and resources within
-                        that workspace will be permanently removed and cannot be recovered.
-                      </span>
-                      <div>
-                        <Button variant="danger" onClick={() => setDeleteWorkspaceModal(true)}>
-                          Delete my workspace
-                        </Button>
-                      </div>
-                    </div>
-                  </Disclosure.Panel>
-                </Transition>
-              </div>
-            )}
-          </Disclosure>
-        )}
+        {isAdmin && <DeleteWorkspace workspace={currentWorkspace} />}
```

Comment:
Give it a better name. DeleteWorkSpaceSection, maybe.

Reply:
Done!